.github/workflows/scripts/scanner-fetch-nvd.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+base_url="https://services.nvd.nist.gov/rest/json/cves/2.0"
+results_per_page=2000
```

Comment:
NVD Api has constraint to fetch data at maximum 2000 CVEs per request.

Reply:
I think it's worth adding a comment about that in the script.